Sources/MongoSwift/MongoSwift.swift

```diff
@@ -12,18 +12,6 @@ private final class MongocInitializer {
     }
 }
 
-/// :nodoc:
-@available(*, deprecated, message: "Calling this method no longer has any effect.")
```

Comment:
fortunately happened to catch that these were still in the codebase. I searched and didn't find anything else deprecated still around.